### PR TITLE
remove bug in local development docker compose api script

### PR DESCRIPTION
**Pull request by naomatheus**
## summary

Removes a line in the build scripts for the API container in docker-compose build configuration.

### CODE REVIEW

**Comment by naomatheus** [comment]
@leothomas 
Hey you were right Leo, this `mypy` doesn't need to be there and actually was causing issues in local development.
Could you merge this when you get a chance?

**Review by leothomas — APPROVED** [review]
Looks good to me! I'll hold off on merging this because its Friday afternoon and I'm not familiar enough with the project anymore to resolve any deployment issues before the weekend!

**Pull request merged**
